packages/storybook/examples/core/SearchInput/BasicSearchInput.js

```diff
@@ -3,22 +3,50 @@ import React from 'react';
 import SearchInput from '@ichef/gypcrete/src/SearchInput';
 import DebugBox from 'utils/DebugBox';
 
-function BasicSearchInputExample() {
-    return (
-        <div>
-            <DebugBox>
-                <SearchInput />
-            </DebugBox>
-
-            <DebugBox>
-                <SearchInput defaultValue="Monkey King" />
-            </DebugBox>
-
-            <DebugBox>
-                <SearchInput defaultValue="Monkey King" status="loading" />
-            </DebugBox>
-        </div>
-    );
-}
+export default class BasicSearchInputExample extends React.Component {
+    state = {
+        controlledInputValue: '',
+    }
+
+    handleSearch = (keyword) => {
+        console.log('handleSearch', keyword);
```

Comment:
You can take advantage of `@storybook/addon-actions` plugin to log to Storybook's lower panel. (And it's already installed!)
Please refer to [here](https://www.npmjs.com/package/@storybook/addon-actions) for details.

Reply:
Thanks!